src/test/features/dashboard/routes/claimant.ts

```diff
@@ -64,8 +64,7 @@ describe('Dashboard - claimant page', () => {
         await request(app)
           .post(claimantPage)
           .set('Cookie', `${cookieName}=ABC`)
-          .expect(res => expect(res).to.be.redirect.toLocation(
-            CCJPaths.theirDetailsPage.uri.replace(':externalId', sampleClaimDraftObj.externalId)))
+          .expect(res => expect(res).to.be.redirect.toLocation(CCJPaths.theirDetailsPage.evaluateUri({ externalId: sampleClaimDraftObj.externalId })))
```

Comment:
is this line shorter than 120 chars?

Reply:
Not sure, I tend to ignore max length restrictions as I think breaking when line break is forced by such constraint it does not make code more readable.

Reply:
1. it makes reviewing harder
2. it makes seeing it in the IDE harder.

I think its worth breaking it on `toLocation`

Reply:
Could have defined sampleClaimDraftObj.externalId as a const but otherwise it looks fine

Reply:
@rattur ?

Reply:
Reverted line break